CMakePresets.json

```diff
@@ -0,0 +1,125 @@
+{
+  "version": 4,
+  "cmakeMinimumRequired": {
+    "major": 3,
+    "minor": 23,
+    "patch": 0
+  },
+  "configurePresets": [
+    {
+      "name": "nexstim",
+      "generator": "Visual Studio 16 2019",
+      "binaryDir": "../BUILD-3.6.7",
+      "cacheVariables": {
+        "BUILD_APPS": {
+          "type": "BOOL",
+          "value": "OFF"
+        },
+        "BUILD_SHARED_LIBS": {
+          "type": "BOOL",
+          "value": "OFF"
+        },
+        "BUILD_SINGLE_SHARED_LIBRARY": {
+          "type": "BOOL",
+          "value": "OFF"
+        },
+        "CMAKE_CONFIGURATION_TYPES": {
+          "type": "STRING",
+          "value": "Debug;Release;MinSizeRel;RelWithDebInfo;UsableDebug"

```

Comment:
Remove MinSizeRel and RelWithDebInfo [[SNBT-3665](https://jira.nexstim.local:8443/browse/SNBT-3665)].

Also other references later in the file.